content/en/tracing/setup_overview/compatibility_requirements/php.md

```diff
@@ -46,29 +46,33 @@ PHP APM supports the following SAPI's:
 
 By default, Datadog **supports all PHP web frameworks** out of the box, which allows you to see traces for spans of supported libraries—for example: database and HTTP clients.
 
+For a growing set of web frameworks Datadog offers full instrumentation, including internals methods. Other frameworks still benefit from our generic web tracing, until Datadog will instrument the framework's internal methods.
```

Comment:
I think let's be even clearer here and stick with the same terms. 

I'd say:

By default, Datadog supports all PHP web frameworks out of the box, with either Framework-Level Instrumentation(you can say Full if you want here but I think this sounds better), or Generic Web Tracing.

Framework-Level Instrumentation includes tracing of internals methods and framework specific tagging.

Generic web tracing will include a `web.request` span to track latency and errors that originated from the call. 

The following table enumerates some of the frameworks and versions Datadog successfully traces.

Then I'd update the table to use these two terms.

Reply:
I like the idea Andrew. Thanks for the feedback. Changing right away

Reply:
@andrewsouthard1 please review latest changes. Only one very small edit I have done if you agree:

`Generic web tracing will include a `web.request` span to track latency and errors that originated from the call [added], in addition to spans for supported libraries — for example: database and HTTP clients.[\added]`